tests/decorators/decorators.go

```diff
@@ -50,6 +50,7 @@ var (
 	USB                                  = []interface{}{Label("USB")}
 	AutoResourceLimitsGate               = []interface{}{Label("AutoResourceLimitsGate")}
 	RequiresTwoWorkerNodesWithCPUManager = []interface{}{Label("requires-two-worker-nodes-with-cpu-manager")}
+	DualStackCluster                     = []interface{}{Label("dual-stack-cluster")}
```

Comment:
Aren't most of our lanes dual stack? I think the exception is to have a single stack case, no?
How about to place a label like `ipv6-single-stack` and `ipv4-single-stack`?

Reply:
> Aren't most of our lanes dual stack? I think the exception is to have a single stack case, no?
correct 

> How about to place a label like ipv6-single-stack and ipv4-single-stack?

the test that is changed requires dual stack, so the minimal change that is needed is that if we are IPv6 single stack, skip dual stack tests
so we do need the label to be dual-stack, not x-single-stack
(PS note that we don't have a ipv4-single-stack kubevirtci variant)

Reply:
This is super confusing.

Then perhaps this name will fit better: `requires-dual-ip-stack-cluster`.

Reply:
done with `requires-dual-stack-cluster ` hope its fine